Project4/style.css

```
@@ -0,0 +1,637 @@
+    background-image: -o-linear-gradient(top, #eeeeee, #fcfcfc);
+    background-image: linear-gradient(to bottom, #eeeeee, #fcfcfc);
+    -webkit-box-shadow: inset 0 1px rgba(0, 0, 0, 0.03);
+    box-shadow: inset 0 1px rgba(0, 0, 0, 0.03);
+}
+
+.register-button:focus {
+    outline: 0;
+}
+
+.register-switch {
+    height: 32px;
+    margin-bottom: 15px;
+    padding: 4px;
+    background: #6db244;
+    border-radius: 2px;
+    background-image: -webkit-linear-gradient(top, #60a83a, #7dbe52);
+    background-image: -moz-linear-gradient(top, #60a83a, #7dbe52);
+    background-image: -o-linear-gradient(top, #60a83a, #7dbe52);
+    background-image: linear-gradient(to bottom, #60a83a, #7dbe52);
+    -webkit-box-shadow: inset 0 1px rgba(0, 0, 0, 0.05), inset 1px 0 rgba(0, 0, 0, 0.02), inset -1px 0 rgba(0, 0, 0, 0.02);
+    box-shadow: inset 0 1px rgba(0, 0, 0, 0.05), inset 1px 0 rgba(0, 0, 0, 0.02), inset -1px 0 rgba(0, 0, 0, 0.02);
+}
+
+.register-switch-input {
+    display: none;
+}
+
+.register-switch-label {
+    float: left;
+    width: 50%;
+    line-height: 32px;
+    color: white;
+    text-align: center;
+    text-shadow: 0 -1px rgba(0, 0, 0, 0.2);
+    cursor: pointer;
+}
+
+.register-switch-input:checked+.register-switch-label {
+    font-weight: 500;
+    color: #434248;
+    text-shadow: 0 1px rgba(255, 255, 255, 0.5);
+    border-radius: 2px;
+    -webkit-box-shadow: 0 1px 1px rgba(0, 0, 0, 0.15), 0 0 0 1px rgba(0, 0, 0, 0.1);
+    box-shadow: 0 1px 1px rgba(0, 0, 0, 0.15), 0 0 0 1px rgba(0, 0, 0, 0.1);
+}
+
+:-moz-placeholder {
+    color: #aaa;
+    font-weight: 300;
+}
+
+::-moz-placeholder {
+    color: #aaa;
+    font-weight: 300;
+    opacity: 1;
+}
+
+::-webkit-input-placeholder {
+    color: #aaa;
+    font-weight: 300;
+}
+
+:-ms-input-placeholder {
+    color: #aaa;
+    font-weight: 300;
+}
+
+::-moz-focus-inner {
+    border: 0;
+    padding: 0;
+}
+
+
+h1 {
+    font-family: 'Orbitron', sans-serif;
+
+    font-size: 45px;
+}
+
+h3 {
+    font-family: 'Gugi', cursive;
+    font-size: 50px;
+    color: #f0ebeb;
+}
+
+h4 {
+    font-family: 'Teko', sans-serif;
+}
+
+p {
+    font-family: 'Khand',
+        sans-serif;
+    font-size: 20px;
+    color: #cccccc;
+
+}
+
+.mt medium diagonal {
+    align-items: center;
+}
+
+@media (min-width: 375px) {
+    .mt medium diagonal {
+        align-items: center;
+    }
+
+}
+
+.overlay-item {
+    display: block;
+    position: relative;
+    overflow: hidden;
+    text-align: center;
+
+}
+
+.overlay-item .mask {
+    display: block;
+    position: absolute;
+    overflow: hidden;
+    top: 0;
+    left: 0;
+    height: 100%;
+}
+
+.overlay-item img {
+    display: block;
+    width: 100%;
+    height: auto;
+    x-margin-right: auto;
+    x-margin-left: auto;
+}
+
+.overlay-item h3 {
+    text-transform: uppercase;
+    text-align: center;
+    position: relative;
+    padding: 10px;
+    margin: 0px 0px 20px 0px;
```

Zero measurements do not need units:
```suggestion
    margin: 0 0 20 0;
```

Also, since headings only have top and bottom margins by default, you could also do this instead:
```suggestion
    margin-top: 0;
    margin-bottom: 20px;
```